crates/beautifier/src/lib.rs

```diff
@@ -0,0 +1,292 @@
+/*
+ * Copyright 2022 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#![deny(
+    dead_code,
+    nonstandard_style,
+    unused_imports,
+    unused_mut,
+    unused_variables,
+    unused_unsafe,
+    unreachable_patterns
+)]
+
+use std::fmt::Display;
+use std::io::{Error as IoError, Result as IoResult, Write};
+
+use air_parser::ast::{
```

Comment:
In this repo we follow this import scheme:
- imports from local project (`crate::`/`super::`) 
- newline
- other imports from non-std
- newline
- imports from std

If there are not so many imports newlines could be omitted.

Reply:
Do we need to import all instructions to the global scope? Maybe just import `air_parser::ast` and then use `ast` as a prefix?